sts_synindex_external.R

```diff
@@ -7,29 +7,30 @@ library(rjson)
 synapser::synLogin(authToken = Sys.getenv('SYNAPSE_AUTH_TOKEN'))
 source('~/recover-parquet-external/params.R')
 
-#### Get STS token for bucket in order to sync to local dir ####
 
-# Get STS credentials
-token <- synapser::synGetStsStorageToken(
-  entity = PARQUET_FOLDER_INTERNAL,
-  permission = "read_only",
-  output_format = "json")
+# Get STS credentials for input data bucket -------------------------------
+token <- 
```

Comment:
Nit: I recommend using a linter for your code: https://lintr.r-lib.org/

Reply:
I am setting up linting using GitHub actions, so this will be something added in a different branch